src/components/panel/panel.spec.js

```diff
@@ -104,6 +104,43 @@ describe('$mdPanel', function() {
     expect(PANEL_EL).not.toExist();
   });
 
+  it('should call onDomAdded if provided when adding the panel to the DOM', function() {
+    var onDomAdded = jasmine.createSpy();
+    var animationHookConfig = angular.extend(DEFAULT_CONFIG, {'onDomAdded': onDomAdded});
+
+    panelRef = $mdPanel.create(animationHookConfig);
+
+    panelRef.attachOnly();
+    expect(onDomAdded).toHaveBeenCalled();
+
+    angular.element(document.querySelector(PANEL_WRAPPER_CLASS)).remove();
```

Comment:
This should be removed in the afterEach already and if not it should be there for all tests.